ui/apps/platform/cypress/helpers/request.js

```diff
@@ -165,6 +165,39 @@ export function interceptAndWatchRequests(routeMatcherMap, staticResponseMap) {
     });
 }
 
+/**
```

Comment:
I added these two helpers as there was not a concise way to do this with our `interceptAnd*` functions that was decoupled from the `visit` behavior.

Reply:
Aha, coupling with `visit` was difficult for me too. Maybe this should supersede the other.